syntax/swift.vim

```diff
@@ -43,6 +44,6 @@ hi def link swiftComment	Comment
 hi def link swiftTodo		Todo
 hi def link swiftFunction	Function
 hi def link swiftProcedure	Function
-hi def link swiftImport	Include
+hi def link swiftImport		Include
```

Comment:
Sorry for not seeing this. For some reasons GitHub didn't notify me of incoming pull requests.

But can you remove the extra-spaces in this line?